lib/pc_api.h

```diff
@@ -289,6 +289,8 @@ void pc_schema_set_dimension(PCSCHEMA *s, PCDIMENSION *d);
 void pc_schema_check_xyzm(PCSCHEMA *s);
 /** Get the width in bytes of a single point in the schema */
 size_t pc_schema_get_size(const PCSCHEMA *s);
+/** Check whether the schemas are similar */
+uint32_t pc_schema_similar(const PCSCHEMA *s1, const PCSCHEMA *s2);
```

Comment:
Changed to `pc_schema_equivalent`. I find it better than "similar", but that may not address your comment, as "equivalent" is similarly "vague" to "similar" :)